manifest.json

```diff
@@ -2,7 +2,7 @@
   "manifest_version": 2,
   "name": "Loadster Recorder",
   "description": "Create load test scripts to run in Loadster.",
-  "version": "2.4",
+  "version": "2.5",
```

Comment:
I wasn't sure about 3.0 here, since we have backwards compatibility

Reply:
I think 2.5 is fine.